server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2966,13 +2966,26 @@ public Void run() {
     }
   }
 
+  private Durability getMincEventDurability(KeyExtent extent) {
+    TableConfiguration conf;
+    if (extent.isMeta() || extent.isRootTablet()) {
```

Comment:
This check is a bit confusing. To me it reads "If the extent is for the metadata table or the root tablet, use the root tablet's configuration. Otherwise, use the metadata configuration." Is there some other setting for user tables that's more appropriate? 

`KeyExtent#isMeta` already returns whether or not it's for the metadata table or the root tablet, so the `|| extent.isRootTablet()` is redundant. However, I think refactoring `KeyExtent#isMeta` is more appropriate to make it more clear about what it is checking.